### ISSUE TITLE
[Feature Pitch] add parachutes to CTF

### ISSUE THREAD

**Issue by bunniewormy** [transhumandesign/kag-base#215]
I suggest adding a new item to the builder shop: a parachute. It would slow down falling and allow you to glide for longer. You could put it in inventory. Something like this: https://gfycat.com/TidyLongBlackbear
Builder/archer version: https://gfycat.com/FragrantClassicDragon

It would allow for some new nice metas; you could jump over bigger distances, fly over small enemy towers with the help of parachutes [incentive for builders to make proper towers], and perhaps make your bomb jumps a little better. Of course, to make it not too op, it probably shouldn't be too cheap. I think 80 coins for a version similar to the one in the gifs would be nice; perhaps even some wood as well.

Now, inb4 'but we have chickens', here are the differences between chickens and parachutes:

- Chickens need to hatch from egg first, while parachutes are bought
- Chickens can breed, parachutes can't
- Chickens are easily killed by enemies, while parachutes can be kept in inventory for longer use; also, easier to steal an enemy's parachute this way, rather than accidentally killing it
- Chickens don't allow for 'infinite' flight. With parachutes, you still fall down, but you can glide at a consistent speed and never start speeding up into the ground; the velocity stays the same.
- Chickens can't be put into inventories, parachutes can 

I think this is something interesting enough to be added to the game, could create some new metas (use parachute, then put into inventory while above enemy to start falling faster stomp him), but it probably wouldn't be a game-breaking addition. Would also make catapults more reliable. Also, you could still counter people using parachutes with the new sky limit; you would just need to make a tower that is tall enough. Also, imagine two people with a parachute meeting midair and having an epic battle. :D

**Comment by Franrek** [comment]
So pretty much add op chickens and create cata+parachute meta, no thanks.

**Comment by TheCatWiz** [comment]
No thanks. Chickens are more than enough for official games. However, parachutes would be fun for unofficial servers with big maps.

**Comment by bunniewormy** [comment]
Chickens arent more than enough. They can be killed with a single slash and take like 40 seconds to hatch, you can't put them in inventory, etc. They're used maybe once in 100 matches...

**Comment by eps0003** [comment]
Chickens should be buffed instead of adding a better version of it

**Comment by bunniewormy** [comment]
honestly, I don't really understand why people don't like parachutes. all I've got so far is they would be 'op', and that chickens are 'enough', yet still noone really explained how parachutes would be op. how could they be abused? would people overuse them? is 80 coins too low of an price? u can fly way too far? or is it something else? it's just that anything like that could be changed, so saying something is 'op' doesn't really make sense. or is it something else people don't like about them? is it the fact that we already have chickens that make you fall slower, and people don't want new items? because as I said chickens aren't really good... the last time i remember them being used was on an iko's ctf map.

is them being used together with catapult too OP? because tbh, that's not really what I would've used it for. I think parachutes would be a nice helpful item that would also slightly change some submetas. you could use them instead of a bomb jump to fly to an enemy tower that is small enough to try that. archers flying above mid and shoot arrows at it could be interesting as well. this would just require builders to make better mid bases. you could say that it hurts builders, but builders could also use parachutes: together with knights, they could fly to the top part of enemy towers to attack it from above, instead of having to go bottom and risk getting easily killed. of course, it wouldn't happen all the time; you're not going to spend 80 coins every time you die, right? for what it's worth, parachutes could even cost 100 coins. numbers like that can be changed.

also, in my opinion, you could even remove chickens if you add them. i think chickens are a rather annoying mechanic, with their lack of being put in inventory, having to buy an egg and wait a long time for them to hatch. also, when you were buyign chickens, you would buy them for the team; your teammates would almost always waste them, so even the breeding didn't help a lot. another thing was that chickens were basically useful only when you used them together with catapult; parachutes wouldn't be like that, because they can provide many more possibilities. limiting them to 1 [and a shitty one, by that - #nerfsiege and not just for this, but i'll talk about that another time] meta doesn't make sense. you could even make them unusable together with catapults if you really want, and they could still be used in lots of creative ways to hurt the enemy or just help with mobility.

**Comment by TheCatWiz** [comment]
"u can fly way too far?" "is them being used together with catapult too OP? because tbh, that's not really what I would've used it for. "
yes you can. the parachutes used in homek servers allow you to literally travel 300+ blocks almost without losing height if you are launched from cata. It is even worse than sawjumps. And nobody cares what things are made for, people will always use them in most effective way possible And if you nerf them, then we would have basically the same chickens, but expensive and more portable.
"chickens aren't really good... the last time i remember them being used was on an iko's ctf map."
you should play more KAG, my friend. I still kinda agree with this tho: chickens are really underused, i think hatch time should be reduced, as well as and breeding fix.
" archers flying above mid and shoot arrows at it could be interesting as well"
because we dont have enough waterspam already. archers are way too mobile in comparison to knights even without parachutes.
" you could even remove chickens"
why would we replace existing thing with a new one if we can modify what we have? As i said earlier, buffing chickens would be more than enough.
 "with their lack of being put in inventory"
there is pull request for that. overall most players have nothing against putting chickens in inventory, so it will probably be added soon.
"when you were buyign chickens, you would buy them for the team; your teammates would almost always waste them"
not everything should be personal. KAG is a team game, after all
"limiting them to 1 meta doesn't make sense"
it doesnt make sense, but its what people will do to them, and making them unusable with catas seems like a crutch. also, parachutes would make building tall tower the only working tactic in the game, since knights and builders would easily fly over everything that isnt at sky height, not giving enemies any chance to rebuild. let parachutes stay in modded servers with big maps.

**Comment by FuzzCat927** [comment]
Rather than add parachutes, I think that chicken breeding should be made more consistent so that chicken farms are actually valid. Another thing that makes chickens weak is that your team can easily kill them, whether on purpose or not. Chickens should be teamed.

**Comment by bunniewormy** [comment]
> yes you can. the parachutes used in homek servers allow you to literally travel 300+ blocks almost without losing height if you are launched from cata. It is even worse than sawjumps. And nobody cares what things are made for, people will always use them in most effective way possible And if you nerf them, then we would have basically the same chickens, but expensive and more portable.

Who's saying I want the parachute to be as good on my server? I didn't even say any specific numbers.
'same chickens', except they're actualyl good, you can put them in inventory, and they don't need to hatch. That last part is the worst; as long as you need to wait 1 min, noone is going to buy them.

>"chickens aren't really good... the last time i remember them being used was on an iko's ctf map."
you should play more KAG, my friend. I still kinda agree with this tho: chickens are really underused, i think hatch time should be reduced, as well as and breeding fix.

are you saying im not active enough or what now? nice maymay

>" archers flying above mid and shoot arrows at it could be interesting as well"
because we dont have enough waterspam already. archers are way too mobile in comparison to knights even without parachutes.

maybe blame waterspam on the cheapness of water, instead of mobility?

>" you could even remove chickens"
why would we replace existing thing with a new one if we can modify what we have? As i said earlier, buffing chickens would be more than enough.

because chickens are, uh, animals? not only they need to hatch, but you need to keep them in enclosed space, and you can't even re-use them after getting to the enemy base with them, because they're goign to run away. the only 'buff' to chickens that would work would remove the egg hatching system and make them not move. and make them putable into inventory.

surprise, you have a parachute. parachute that you buy and can put into inventory makes 100x more sense.

> "when you were buyign chickens, you would buy them for the team; your teammates would almost always waste them"
not everything should be personal. KAG is a team game, after all

yeah, but then newbies are going to go fight enemies with a chicken 'cus it's fun' and then that 150 coins you spent to try to make a decent farm for yourself is gone. i'd rather pay 80 coins for 1 chicken that can't breed, can't move and that can be put into inventory than 30 coins for 1 egg. you know why? because then it's a parachute.

back to 'team game' - it's a good argument, but it shouldn't apply to items that help with mobility. team sharing resources in storage? alright, but shared items kinda suck. would you be fine with having to give away 1/2 of the bombs you buy?

>"limiting them to 1 meta doesn't make sense"
it doesnt make sense, but its what people will do to them, and making them unusable with catas seems like a crutch. also, parachutes would make building tall tower the only working tactic in the game, since knights and builders would easily fly over everything that isnt at sky height, not giving enemies any chance to rebuild. let parachutes stay in modded servers with big maps.

i'd say you're hugely exaggerating. how would it be different from people using chickens to fly over everything? and no, you don't need towers to be as tall as sky height. in the end, this could as well be a 'buff chicken' pitch, except then the behavior of chickens wouldn't make sense, hence parachutes.

parachutes are just literally chickens without their annoying behaviour and inconsistency


tl;dr:

change chicken sprite into parachute
make chickens buyable without the need for an egg
make chickens putable into inventory
make chickens not move
make chickens not breed
chicken price 80 coins
give them actual health so they can't be instakilled with an accidental jab

i still fail to see how all of this would be too OP

**Comment by TheCatWiz** [comment]
> Who's saying I want the parachute to be as good on my server? I didn't even say any specific numbers.
Your example gifs showed parachutes from homek servers, so i assumed you wanted parachutes to be almost like these OP things

> are you saying im not active enough or what now?
not really. i wanted to say that people still sometimes use them on big maps or to end stalemates.

> maybe blame waterspam on the cheapness of water, instead of mobility?
cheapness of water is another topic, but mobility parachutes give would allow archers to hang in a middle of nowhere, bombarding everything and everyone with water or even bombarrows, being able to fly back whenever they want.

>  not only they need to hatch, but you need to keep them in enclosed space, and you can't even re-use them after getting to the enemy base with them, because they're goign to run away.
These things are the only reason why chickens aren't overused as hell. Give few good knights a catapult at the base and parachutes, and they would just hang around enemy base, farming money for more parachutes and mines/bombs, making repairing towers almost impossible for enemy team, because even if they will kill a knight, he will come back 15 seconds later. And if there are 2-3 knights like this, then its gg. Having kind of "cooldown" for chickens is really needed thing.

>  how would it be different from people using chickens to fly over everything?
Already described that in previous statement. Of course, you can buy eggs every minute so you could always have a chicken ready, but usually people dont control those things.

>  team sharing resources in storage? alright, but shared items kinda suck. would you be fine with having to give away 1/2 of the bombs you buy?
If we will be able to put chickens in inventory, we will be able to put them into storages as well, and most newbies never check them.

> and no, you don't need towers to be as tall as sky height
you do, because even usual chickens allow you to fly ~70 blocks losing only 3-4 blocks height, so on most official maps, you can fly from home base right to enemies if you tower is max height.

> i still fail to see how all of this would be too OP
And i still fail to see why we need to fully remove chickens. Half their hatch time, fix breeding, make them cost maybe 40-50 coins, make them jump around less, give them some more HP.
This dispute leads to nowhere, because both of us will keep their opinion.

**Comment by Suprejso** [comment]
Parachutes were a really nice feature on HOMEK vs all and Homek Big Battle CTF. Though they were needed because maps there were big, sometimes huge. I don't think it would really work out on vanilla CTF though.
Not saying the idea is bad, don't misunderstand me, if the parachutes weren't as effective as in the servers i mentioned earlier I'd definitely appreciate it.

**Comment by Basea** [comment]
I don't see a need of adding parachutes to official servers as well, they're not balanced well, even if you reduce their fly-time, they still are going to be overpowered, it may destroy the balance, which is already easy destroyable by some of that pitches.
Chickens are underrated by players, they should be buffed, tho.
Am using them in game, chickens gives an advantage for knights to defend.
You want your item to be added, while it doesn't have defects, except the breeding thing, and may make the game even more difficult for newbies.
Also, try to imagine, how it will affect the competitive part of the game.
Probably, I don't see any advantages for KAG from adding parachutes, but apparently, it's not a necessary update.

**Comment by bunniewormy** [comment]
ITT: 'parachute in gif too op, shit suggestion'
'parachute too op even if u nerf fly time, btw i have no arguments, may destroy balance, but idk how'
'b-b-but chickens???'
im fine with parachutes having defects, but the chickens defects are just annoying and shit, which limits them a lot. btw, if youre using chickens in combat while there's a catapult, you're wasting them [only reason for chickens is catapult]. a good defect could be the parachute being able to be shot down with 1 arrow, or having a limited glide amount [works for lets say 15-30 seconds, after that you need to touch the ground]. 
it won't make the game more difficult for newbies. newbies struggle with the mechanics being hard to learn [movement, fighting properly]. they won't struggle with metas like that, because you can learn that from being observant; you just need to learn the basics [shops, building] before you can start doing more 'advanced' stuff. meanwhile, fighting and combat is mainly muscle memory, reflex, and knowledge of how the physics in the game work [something that can be learned most easily by having someone else explain, or reading a tutorial] - fighting is pressing buttons, metas like knowing when to do siege or when to use an item is just thinking. and how would it even make the game more difficult? more difficult that they will have to learn how this item works [while theres a chicken that makes even less sense] or more difficult playing against enemies which use it? because there are still catapults, bomb jumps and sawjumps to travel bigger distances/get to the enemy base. i want a new item to be added to just make things more interesting and perhaps introduce a new submeta. on most maps parachutes wont even be an OP thing, with all the shitmaps with low sky or stuff like that. 
no need to worry for the competitive part of the game (which you dont play)

**Comment by Mazey** [comment]
I don’t see a point in adding this, especially as a paid bounty. You can use chickens, catapults and crates.

Chickens have problems, yes, but you can use catapult w/crate then. There isn’t a place for “perfect” (not OP) gliders/parachutes.

**Comment by Basea** [comment]
"no need to worry for the competitive part of the game (which you dont play)"
Such a shit argument.
It does make game harded for newbies, who gets more advantage by parachutes?
A good player, who know how to use it, or a newbie?
It makes pub-stomp by good players easier, there is only two ways for updates like this, they'll be OP or they won't be used at all, you can't balance it well and they'll repeat quarry's history(As you can see, quarries aren't used in any type of games).

**Comment by bunniewormy** [comment]
you know that we coudl just do a vote in the competitive community and decide if competitive should have parachutes or not? it's literally as simple as that
yes, you're saying that good players are better because they have experience, that's how this game works. newbs are going to get always stomped by good players, no matter the technique used

**Comment by Basea** [comment]
It's going to be an imbalance addition for all classes, good builder with parachute can win match solo, that's what I meant, show me a balanced version of parachute, so I may change my opinion.

**Comment by bunniewormy** [comment]
you know that parachutes are going to be the cost of a catapult, right? he can 'win match solo' only if the entire team ignores him for like 5 minutes, stop exaggerating. he could do the same if he was launched with a catapult [that has the same price] and had a crate with him...

there are many more 'imbalance' OP ways to 'win match solo' than parachutes. knight sawjump with crate with builder is one example. this addition wouldnt even change that much, just make some stuff more interesting.

**Comment by Basea** [comment]
It's going to be a good innovation, if you'll balance it without removing the main things that makes it not the same as the chicken.

**Comment by dpants** [comment]
I love the idea. Its just a small change but could make for a really fun time.

**Comment by bunniewormy** [comment]
closing since it wasn't liked by the community and now upvoted enough on #democracy
